src/views/Login/AuthButton.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../Button";
+import { Logout } from "../../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onLogout = this.onLogout.bind(this);
```

Comment:
If you change the `onLogout()` to an ES6 function you will not need the bind statement nor the constructor. Read more here https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56?gi=8207f4fed723

Reply:
@kimotho-njoki Noted